#
# old_revision [dba36543cdde6db127857b0edfdc3ad1969bbd39]
#
# patch "libpurple/protocols/oscar/oscar.h"
#  from [a40e3332eade975c73e344e17bd06d4b99cae751]
#    to [7853a27f5f8735092ebefef567aaeeccad2a057c]
#
============================================================
--- pidgin/libpurple/protocols/oscar/oscar.h	a40e3332eade975c73e344e17bd06d4b99cae751
+++ pidgin/libpurple/protocols/oscar/oscar.h	7853a27f5f8735092ebefef567aaeeccad2a057c
@@ -301,7 +301,7 @@ struct _ClientInfo
 
 #define CLIENTINFO_PURPLE_ICQ { \
 	"Purple/" VERSION, \
-	0x010a, \
+	0x010b, \
 	0x0014, 0x0034, \
 	0x0000, 0x0bb8, \
 	0x0000043d, \
